routers/user.controller.js

```diff
@@ -122,7 +124,9 @@ class UserController {
 
   async getCurrentUser(req, res) {
     try {
-      const { _id, name, email, familyId } = req.user;
+      const { _id, name, email } = req.user;
+
+      const familyId = await userModel.findOne({ _id }).populate('familyId');
```

Comment:
you need only to find family by id from a family model, here this operation is too complex and overmuch. And after find operation you are not getting a familyId, but a whole family, so name it as is. And don't forget to do changes in response like with _id.